src/main/java/com/cburch/logisim/gui/start/TtyInterface.java

```diff
@@ -321,9 +325,9 @@ public static void run(Startup args) {
 
     // we load the ram before first propagation
     // so the first propagation emits correct values
-    if (args.getLoadFile() != null) {
+    for (Pair<File, String> p : args.getLoadMems()) {
```

Comment:
`final var` please

Reply:
Happy to use a final.

However, not sure about the var here since a pair is used and the code is much more readable by explicitly stating the type of the pair to avoid any confusion when calling getLeft()/getRight() later. 

Of course since this is my first contribution to the code base, I will not argue any further if you disagree.